src/nv_ingest/schemas/vdb_task_sink_schema.py

```diff
@@ -37,10 +37,11 @@ def build_default_milvus_config(embedding_size: int = 1024) -> typing.Dict[str,
         "index_conf": {
             "field_name": "vector",
             "metric_type": "L2",
-            "index_type": "HNSW",
+            "index_type": "GPU_CAGRA",
```

Comment:
I am thinking it is ok to expect GPU support by default, given that the milvus image is also GPU specific. Let me know if we need to be more malleable in this regard.